.github/workflows/build_dockerfiles.yml

```diff
@@ -0,0 +1,36 @@
+name: Check dockerfiles
+
+on: push
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Build Ubuntu 14.04
+        run: docker build --rm --no-cache -t litmusimage/ubuntu:14.04 . -f apt_initd_dockerfile --build-arg BASE_IMAGE_TAG=14.04 --build-arg OS_TYPE=ubuntu
```

Comment:
`litmusimage` vs `litmus_image` ? we should stay with one spelling - I don't care which one.

Reply:
dockerhub does not accept underscores in org names... :(